app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -3,7 +3,6 @@ title: Subject knowledge enhancement (SKE)
 related_content:
     Get school experience : "/train-to-be-a-teacher/get-school-experience"
     Find teacher training courses : "https://www.gov.uk/find-postgraduate-teacher-training-courses"
-    Application tips : "/blog/application-tips-from-a-teacher-training-provider"
     Engineers teach physics : "/subjects/engineers-teach-physics"
```

Comment:
Should we change this to link to the general physics page rather than ETP @EllieNodder @philspray ?